### ISSUE TITLE
Flaky a11y smoke test in CI

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#286804]
Seeing 

```
 1) VSCode Smoke Tests (Web)
       Accessibility
         Workbench
           workbench has no accessibility violations:
     Error: Accessibility violations found:

[aria-allowed-attr] Elements must only use supported ARIA attributes (critical)
  Help URL: https://dequeuniversity.com/rules/axe/4.11/aria-allowed-attr?application=axeAPI
  Element: .list_id_5
    Class: monaco-list list_id_...
    HTML: <div class="monaco-list list_id_..." tabindex="0" role="list" aria-label="Error: Activating ex..." aria-multiselectable="true" data-keybinding-cont...="17">
    Issue: Fix all of the following:
  ARIA attribute is not allowed: aria-multiselectable="true"

  Element: #list_id_5_0
    Class: monaco-list-row
    HTML: <div class="monaco-list-row" role="dialog" data-index="0" data-last-element="true" data-parity="even" aria-setsize="1" aria-posinset="1" id="list_id_5_0" aria-selected="false" aria-label="Error: Activating ex..." draggable="false" style="top: 0px; height: 42...">
    Issue: Fix all of the following:
  ARIA attributes are not allowed: aria-setsize="1", aria-posinset="1", aria-selected="false"

```
